tripy/tripy/backend/mlir/memref.py

```diff
@@ -0,0 +1,105 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 1993-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from tripy.backend.mlir import utils as mlir_utils
+from tripy.common import device as tp_device
+from tripy.common import utils as common_utils
+
+import mlir_tensorrt.runtime.api as runtime
+
+RUNTIME_CLIENT = mlir_utils.MLIRRuntimeClient()
```

Comment:
Isn't this already a singleton?

Reply:
Yes, should I just replace all `RUNTIME_CLIENT` with `mlir_utils.MLIRRuntimeClient()` then?

Reply:
Updated.